src/mon/Monitor.h

```diff
@@ -1024,6 +1044,9 @@ class Monitor : public Dispatcher,
 
   OpTracker op_tracker;
 
+  MonitorBackupManager *backup_manager;
```

Comment:
Why is this a pointer and not just a directly contained value?  If you do actually need an indirection, please use a std::unique_ptr.